### PR TITLE
maven-artifact-choicelistprovider-plugin

### PR DESCRIPTION
**Pull request by phreakadelle**
Please Review

### CODE REVIEW

File: permissions/plugin-maven-artifact-choicelistprovider.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "maven-artifact-choicelistprovider"
+paths:
+- "org/jenkins-ci/plugins/maven-artifact-choicelistprovider"
```
Comment (daniel-beck) [inline permissions/plugin-maven-artifact-choicelistprovider.yml]: Wrong groupId: https://github.com/jenkinsci/maven-artifact-choicelistprovider-plugin/blob/master/pom.xml#L13

**Comment by daniel-beck** [comment]
Other than groupId mismatch with pom, looks good.

File: permissions/plugin-maven-artifact-choicelistprovider.yml (hunk 1 above)
Comment (phreakadelle) [inline permissions/plugin-maven-artifact-choicelistprovider.yml]: Okay. Pushed my changes to the jenkins-fork and now the groupId should match the groupId in this push request.

Comment (phreakadelle) [inline permissions/plugin-maven-artifact-choicelistprovider.yml]: @daniel-beck : Do you know, why i cant see my plugin on the jenkins http://jenkins.ci.cloudbees.com ?

**Pull request merged**